Project/BookDetail/Comments/CommentListViewModel.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  CommentListViewModel.swift
+//  Project
+//
+//  Created by Emmanuel Velez Agudelo on 11/13/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Foundation
+import WolmoCore
+
+import ReactiveSwift
+import Result
+class CommentListViewModel  {
+    private let _comments = MutableProperty<[CommentViewModel]>([])
+    private var currentPage : Int  = 1
+
+    public let comments:Property<[CommentViewModel]>
+    public let booksRepository : BooksRepository
+    
+    init(booksRepository : BooksRepository = NetworkingBootstrapper.shared.createBooksRepository() ) {
+        comments = Property(_comments)
+        self.booksRepository = booksRepository
+    }
+    public func loadComments(bookId: Int){
+        booksRepository.fetchBookComments(bookId: bookId)
+            .map {
+                $0.map {
+                    CommentViewModel(comment: $0)
+                }
+            }
```

Comment:
You can go like this here: `.map { $0.map { CommentViewModel(comment: $0) } }`
This way, even if it's kinda less readable, you prioritize the structure of your higher order functions.